src/nexus.h

```diff
@@ -89,7 +98,7 @@ public slots:
     ~Nexus();
 
 private:
-    Profile* profile;
+    Profile* mProfile;
```

Comment:
Well, how would you read the following (valid) code?

```c++
Game::Game(Ball* ball)
{
    ball = ball; // "noop" (even more: this->ball is not initialized!)
}
```

…and so does the compiler! :smile:

To differenciate, the name must be different or scoped. Scoping however includes a high risk of failure. Thus it is done by prefixing a member, because it is a) clear, b) less error prone and c) can be used in c'tor initializer lists:

```c++
// works
mProfile = profile;

// also works
this->profile = profile;

// doesn't work
SomeClass(Ptr* profile) 
: this->profile(profile)
{
}
```

The question is: "Why qTox decided against a prefix in class members?" (This is a recommended practice in OOP.)